src/treemodel/range.js

```diff
@@ -25,33 +25,41 @@ CKEDITOR.define( [ 'treemodel/position', 'treemodel/positioniterator', 'utils' ]
 			 *
 			 * @property {treeModel.Position}
 			 */
-			this.start = start;
+			this.start = Position.createFromPosition( start );
```

Comment:
It means that if you create a static range from the live position the live position is transformed into static position? This should be mentioned in the range class or constructor documentation.